packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -34,7 +38,11 @@ export const Default = {
 
 export const HeaderInteractions = {
   args: {
-    columns: ["name", "age", "birthday"],
+    columns: [
```

Comment:
Instead of duplicating it, we could use `Default.args.columns` to re-use them from the default story